_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -779,20 +786,31 @@ sensitive machine user access keys.
 === CloudTrail
 
 You'll want to enable CloudTrail in every single AWS account so that you have an audit log of the major activity
-happening in the account. We typically recommend creating an S3 bucket in the security account and sending all the
-CloudTrail logs from the other accounts to this one S3 bucket. Also, make sure to encrypt all logs with KMS, and only
-give a small number of trusted admins access to the KMS master key and the S3 bucket. You may also want to send the
-logs to CloudWatch Logs a second way to store/view audit logs.
+happening in the account. We typically recommend that you aggregate these logs in the logs account. To do this, you:
+
+. Create an S3 bucket in the logs account and grant the CloudTrail service in all accounts write access to this
+  bucket.
+. Create a KMS Customer Master Key (CMK) in the logs account and grant the CloudTrail service in all accounts
+  permissions to encrypt data with this CMK.
+. Enable CloudTrail in all accounts, configuring it to encrypt data with the CMK in the logs account and write data
+  to the S3 bucket in the logs account.
+. (Optional) Configure CloudTrail to also send logs to CloudWatch Logs within each account. This gives you an
+  additional way to quickly view recent CloudTrail data for an account in the account itself, without having to login
+  to the logs account.
 
 === AWS Config
 
 You'll want to enable AWS Config in every one of your AWS accounts, plus any AWS Config Rules that are relevant to your
 company, so that you can track AWS resource configuration changes over time and enforce company policy. Note that you
 should enable AWS Config in every AWS region, and not just the region(s) you're using for the rest of your infrastructure.
 That way, if an employee (perhaps accidentally), or even worse, an attacker, runs something in a region you don't typically
-use, it'll still be recorded in AWS Config, and your rules will still be enforced. We typically recommend creating a separate
-S3 bucket for each region and storing all of the configuration items there for a set number of days, after which they should
-be moved to Glacier.
+use, it'll still be recorded in AWS Config, and your rules will still be enforced.
+
+We typically recommend that you aggregate AWS Config data in the logs account. To do this you:
```

Comment:
NIT: Only for consistency with [line 789](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/344/files#diff-1bef727ea530a0297ea01495be18648eR789).
```suggestion
We typically recommend that you aggregate AWS Config data in the logs account. To do this, you:
```